source/manage-files.rst

```diff
@@ -0,0 +1,191 @@
+=======================
+Manage Data: Files View
+=======================
+
+To manage the files in a Pod, go to :guilabel:`Files` view in
+PodBrowser. The :guilabel:`Files` view is the default view, but you can
+also switch to the :guilabel:`Files` view by clicking on the Files icon
+on the top menu bar.
+
+.. image:: /images/podbrowser-files-view.png
+   :alt: PodBrowser Files View
+   :width: 600px
+
+The :guilabel:`Files` view displays the contents of your Pod.
+The contents of your Pod have the following type:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 80
+
+   * - Type
+
+     - Description
+
+   * - Container
+
+     - A Container is analogous to a folder; it contains other
+       Containers and Resources.
+
+   * - Resource
+
+     - A Resource refers to a file of any type.
+
+Navigate the Pod
+================
+
+In the :guilabel:`Files` view, click on a folder to navigate
+into the folder.
+
```

Comment:
If you want to navigate directly to a Container in your Pod, or even in another Pod:

#. Click the Pod Indicator dropdown just below the Account menu.

#. Select the :guilabel:`Change Pod` option.

#. Enter the full URL of the Container to which you want to navigate (e.g., `https://<Pod URL/public`).

#. Click :guilabel:`GO`

The contents of the Container will be displayed if you have Read access.